default/stringtables/en.txt

```diff
@@ -6025,18 +6046,46 @@ A planet focused on Growth extends those benefits to other planets connected to
 
 [[GROWTH_SPECIALS_ENTRY_LIST]]'''
 
+INDUSTRY_FOCUS_TITLE
+Industry Focus
+
+INDUSTRY_FOCUS_TEXT
+The Industry Focus represents a plantey primarily focused on industrial production. [[metertype METER_INDUSTRY]] will receive a bonus which increases as the planetary population increases. This is the default focus for any species that does not have a preferred focus.
+
 INFLUENCE_FOCUS_TITLE
 Influence Focus
 
 INFLUENCE_FOCUS_TEXT
 Influence isn't implemented yet.
 
+LOGISTICS_FOCUS_TITLE
+Logistics Focus
+
+LOGISTICS_FOCUS_TEXT
+By focusing on Logistics, a planet is able to vastly extend its supply. The [[metertype METER_SUPPLY]] will be increased by a factor of three.
+
+PLANETDRIVE_FOCUS_TITLE
+Planet Drive Focus
+
+PLANETDRIVE_FOCUS_TEXT
+After a [[BLD_PLANET_DRIVE]] has been constructed, and a [[buildingtype BLD_PLANET_BEACON]] or [[shippart SP_PLANET_BEACON]] is in an adjacent system, selecting the [[PLANETDRIVE_FOCUS_TITLE]] allows the planet to move between star systems.
```

Comment:
probably [[buildingtype BLD_PLANET_DRIVE]]

[[PLANETDRIVE_FOCUS_TITLE]] focus

allows the planet to move -> will move the planet